client/src/store/historyStore/model/watchHistory.js

```diff
@@ -21,6 +21,16 @@ let lastUpdateTime = null;
 // last time changed history items have been requested
 let lastRequestDate = new Date();
 
+document.addEventListener("visibilitychange", function () {
+    if (document.visibilityState === "visible") {
+        // Poll every 3 seconds when visible
+        throttlePeriod = 3000;
```

Comment:
```suggestion
        throttlePeriod = 3000;
        rewatchHistory();
```
?

Reply:
Yep; that's what I had in mind with the todo.  Added with the tweak I made to delay listener kickoff.